modular_skyrat/modules/emotes/code/dna_screams.dm

```diff
@@ -0,0 +1,93 @@
+/datum/species
+	var/list/screamsounds = list('modular_skyrat/modules/emotes/sound/voice/scream_m1.ogg', 'modular_skyrat/modules/emotes/sound/voice/scream_m2.ogg')
+	var/list/femalescreamsounds = list()
```

Comment:
If most species get the same female screams, wouldn't it be better to make them inherit it?

Reply:
No because some of them have male and female screams be the same. Makes it easier.